classes/constellation.php

```diff
@@ -50,15 +50,87 @@ public function render_incidents($future=false, $offset=0, $limit = 5, $admin =
     $show = !$future && $incidents["more"];
 
     $offset += $limit;
-
+    $time_reference = new DateTime('NOW');
+    $date_previous_stored = null;
+    $date_incident_previous = null;
+    $incident_header_date = null;
+
+    // Check to see if call is coming from the admin page. If loaded from the admin page
+    // the date headers will not be rendered.
+    // TODO: Should be an eaiser way to determine this...
+    $is_on_admin = false;
```

Comment:
Is this really needed? I already pass the function `$admin` variable which indicates this :slightly_smiling_face:

Reply:
This is a workaround for the fact that the $admin variable is not set when admin users clicks on "Load more".

Reply:
Oh in that case nice catch. Does that mean that loaded incidents do not have the delete icon?

Reply:
Yes, the delete icon is not available on any records fetched via "Load more".

Reply:
Oh, I will fix that then 🙂